ls1intum/Artemis
# `General`: Fix multiple small type errors in the client application

### Checklist
#### General
#### Client

### Motivation and Context
This PR fixes 3 undefined issues reported on sentry. 

### Description
1) After creating a new text or modeling exercise, no example submissions are present. In this case `exampleSubmissions` is undefined, which lead to an error before.  We'll now use an empty array.
2) Quiz scoring explanation: `submittedAnswers` on a quiz submission can be undefined. We'll use 0 as default value when trying to evaluate the length.
3) Exercise scores chart: `model` can be an empty array, we now check the length to be truthy (>= 1) before accessing `model[0]`.

### Steps for Testing
Code Review should be enough.
You can open the touched components and make sure there are no more errors in the console.

### Review Progress

#### Code Review
#### Manual Tests
